gammapy/data/event_list.py

```diff
@@ -146,6 +146,16 @@ def time(self):
         met = Quantity(self.table['TIME'].astype('float64'), 'second')
         return self.time_ref + met
 
+    @property
+    def obs_start(self):
```

Comment:
I'd suggest to rename to `observation_time_start` and `observation_time_end` for now, for consistency with the other observation-related properties: http://docs.gammapy.org/dev/api/gammapy.data.EventList.html